### DISCUSSION TITLE
Inconsistent use of (A_B)_(C_D)del syntax.

### DISCUSSION THREAD

**Discussion by ifokkema** [HGVSnomenclature/hgvs-nomenclature#133] [HGVS nomenclature proposals]
Related to:
https://groups.google.com/g/hgvs-nomenclature/c/yT7e0Vd7LVw

There is an inconsistency in the description of deletions with unknown breakpoints.
The [deletion page](https://hgvs-nomenclature.org/stable/recommendations/DNA/deletion/) claims `c.(A_B)_(C_D)del` descriptions state that A and D are not deleted. However, for whole-exon deletions with unknown breakpoints in the introns, the format `c.(99+1_100-1)_(200+1_201-1)del` is used. According to the rules, this means the user has confirmed c.99+1 and c.201-1 are present. That is normally not the case, so `c.(99_100)_(200_201)del` would be more logical. Also, when the latter description is projected to the genome, the genomic positions will actually correspond to the exon starts and ends, which also makes more sense.

**Comment by jtdendunnen** [comment]
Officially the description should be based on the assay performed (PCR, qPCR, array, MLPA, ...) and the probes/sequences used. Since many users do not know/can not easily find these and their genomic positions, and since en "exon-based" description format was already used before HGVS nomenclature recommendations were defined we have accepted to allow "exon-based" descriptions of deletions/duplications. The reason to use the format c.(99+1_100-1)_(200+1_201-1)del, and not c.(99_100)_(200_201)del is to make clear the variant break points are most probably located in an intron. From "99_100" I can not "see" these nucleotides are separated by an intron.
The DMD gene variant database (www.LOVD.nl/DMD) contains thousands of examples of descriptions which are probe based, e.g. for MLPA NM_004006.2:c.(6381_6537)_(9245_9338)del for an exon 45-63 deletion.
